### PR TITLE
Update MainActivity.java

### PR DESCRIPTION
**Pull request by AnnanLao**

### CODE REVIEW

**Comment by platisd** [comment]
What does this pull request introduce or change and what issue does it solve?

File: app/app/src/main/java/com/example/myapplication/MainActivity.java [hunk 1]
```diff
@@ -45,8 +45,8 @@
     Connector Headset = new Connector();
 
     boolean eegActive = false;
-    boolean carIsConnected = false;
-    boolean headsetIsConnected = false;
+    boolean carIsConnected;
```
Comment (livmari) [inline app/app/src/main/java/com/example/myapplication/MainActivity.java:48]: This is false by default because the car is not connected by default.

File: app/app/src/main/java/com/example/myapplication/MainActivity.java [hunk 2]
```diff
@@ -45,8 +45,8 @@
...
+    boolean headsetIsConnected;
```
Comment (livmari) [inline app/app/src/main/java/com/example/myapplication/MainActivity.java:49]: This is false by default because the headset is not connected by default.

**Review by livmari — CHANGES_REQUESTED** [review]

**Comment by livmari** [comment]
@AnnanLao Can you name future commits something other than "Update ..."? It is not clear what you changed with that naming :)

File: app/app/src/main/res/layout/content_header.xml [hunk 3]
```diff
@@ -20,6 +20,14 @@
         android:layout_alignParentRight="true"
         android:onClick="goToUrl"
         style="@style/GithubBtn" />
+    
+   <!-- Switch for theme changes-->
```
Comment (livmari) [inline app/app/src/main/res/layout/content_header.xml:24]: Why is this here and not in the dark mode branch merge request?

File: app/app/src/main/java/com/example/myapplication/MainActivity.java (hunk 2 above)
Comment (AnnanLao) [reply app/app/src/main/java/com/example/myapplication/MainActivity.java:49]: It is only for testing between me and Simon. Did not intend to merge it.
